app/javascript/components/supplemental_application/sections/preferences/LiveOrWorkInSanFranciscoPanel.js

```diff
@@ -8,15 +8,15 @@ import formOptions from '~/components/applications/application_form/formOptions'
 import { buildFieldId, memberNameFromPref } from '~/components/applications/application_form/preferences/utils'
 
 const {
-  preferenceProofOptionsDefault,
+  preferenceProofOptionsLiveSf,
   preferenceProofOptionsWorkInSf
 } = formOptions
 
 const individualPreferenceOptions = formUtils.toOptions(['Live in SF', 'Work in SF'])
 
 const getLWPrefProofTypeOptions = (individualPrefName) => {
   if (individualPrefName === 'Live in SF') {
-    return preferenceProofOptionsDefault
+    return preferenceProofOptionsLiveSf
```

Comment:
@akegan I think you worked on this. Any particular reason we were using `preferenceProofOptionsDefault` instead of  `preferenceProofOptionsLiveSf` ?